its/tests/src/test/resources/java-sonarlint.xml

```diff
@@ -22,7 +22,7 @@
       <priority>MAJOR</priority>
     </rule>
 
-    <!-- DEPRECATED keys for 6.7 axis -->
+    <!-- DEPRECATED keys for 7.9 axis -->
```

Comment:
Maybe we could change the comment to say this is to support SonarJava < 6.0 (https://jira.sonarsource.com/browse/SONARJAVA-3160)